### DISCUSSION TITLE
[GENERAL & IDEAS] When Used To Redistribute Credit Is Needed

### DISCUSSION THREAD

**Discussion by MapGuy11** [iptv-org/community#229] [Ideas]
I was scrolling around the internet ,and I see this video: [https://www.youtube.com/watch?v=roWFizjzwXo](https://www.youtube.com/watch?v=roWFizjzwXo) I was suprised how could HE get all these channels so fast! I download the .m3u file and look my specific url's that I post to iptv-org/iptv are on this list I wonder how. Many times in the video he says HE found all these links yeah no it's just a straight copy and paste effort. Also since HE is hosting on his website and not direct linking to the github this list is pretty outdated already. What do you all think? I just get a little annoyed when everyone is praising him for all his hard work when it's all of us that are here day in and day out fixing everything.

**Comment by Sphinxroot** [comment]
sadly there nothing mutch we can do , as soon as its free , its get everywhere  , ive also seen apk linked from here .. free to view with add or buy premium at 2$ for no add .. peoples try to get profit from free stuff  .. not to mention those that ask 3 euro for a stolen list ...

he probably used iptv merge from Mat  to do it , but the main problem , when u dont use the list in remote mode , you don`t get the new stuff or updated links  ...  so , his list will die fast and will need to upload it again and again

i wonder if he get $ from mediafire , if so  .. its questionable ..  its not like the "buy me a coffee " optional donation


but there is a option  that will need a lot of work  ...

have a repo for channel image hosted here 

each image have the repo name in it ex: https://i.imgur.com/HElVWLc.png 

and tvg-logo will have the repo name in it  ex: https://github.com/iptv-org/logos/channels/usa/cnn.png?raw=true

that will force them to get the image and edit the link  .. copy and paste will be reduced a lot

**Reply by Dum4G** [reply]
Trust me, scammers won't care about any watermarks if they decide to raise money. There's nothing we can do and honestly I don't know why should we care for the rest of the web

**Reply by MapGuy11** [reply]
I don't care about scammers I just got really annoyed at how his list is very outdated and how HE says that he put a lot of time into the list.

**Reply by Dum4G** [reply]
Just tell him in comments till it gets removed. Maybe give him a strike on the channel

**Reply by MapGuy11** [reply]
Lol :)

**Comment by jkemp814** [comment]
https://www.youtube.com/watch?v=roWFizjzwXo

https://github.com/iptv-org/iptv/discussions/url

That link routes right back here with 404 from GitHub when you click on it. I had to plain copy, then paste.

**Reply by Sphinxroot** [reply]
my comment will be deleted  but here is https://i.imgur.com/vjECkdn.png

**Reply by MapGuy11** [reply]
Omg!

**Reply by Sphinxroot** [reply]
yup deleted   lol

**Reply by MapGuy11** [reply]
He must know he stole it ,and actively looks for people that call him out.

**Reply by Sphinxroot** [reply]
i reported link as fraud  lets see how YT act

**Reply by MapGuy11** [reply]
Who Knows

**Comment by BellezaEmporium** [comment]
On the contrary, there's also lots of videos (even videos of a sort of bot that scrapes whoever commits in and shows it on a graph) that mentions us and puts our links in the description. And that's cool too. (for example : https://www.youtube.com/watch?v=4hTuibpzHqU (in french))

**Comment by freearhey** [comment]
Just to clarify:

> Anyone is free to copy, modify, publish, use, compile, sell, or
distribute this software, either in source code form or as a compiled
binary, for any purpose, commercial or non-commercial, and by any
means.

The attribution is also not required.

https://github.com/iptv-org/iptv/blob/master/LICENSE
